tests/integration/conftest.py

```diff
@@ -26,7 +26,7 @@
 logging.basicConfig(level="DEBUG" if env.str("DEBUG") else "INFO", stream=sys.stderr)
 logger = logging.getLogger(__name__)
 
-SATELLITE_URL = "satellite.sat.engineering.redhat.com"
```

Comment:
We are actually trying to use the new satellite URL (https://issues.redhat.com/browse/RHELC-732). The dogfood is going to be removed.

Reply:
Yeah, I see. But if you register the system using subscription-manager to get the CentOS 7 satellite server, there are still urls to the dogfood in `/etc/yum.repos.d/redhat.repo`. Maybe adding both of them, so once it would be finally changed, nothing breaks and just be needed to remove the dogfood url from `/etc/dnsmasq.conf`?